styles/thankyou.scss

```diff
@@ -43,6 +43,13 @@
         }
       }
 
+      .icon, img {
+        width: 66px;
```

Comment:
Any reason as to why it specifically 66px x 66px?

Reply:
Yes, it comes from the width and height that we have on Figma.